src/components/scene/editor.js

```diff
@@ -0,0 +1,38 @@
+var registerComponent = require('../../core/component').registerComponent;
+
+module.exports.Component = registerComponent('editor', {
+  schema: { default: true },
+
+  init: function () {
+    this.onKeydown = this.onKeydown.bind(this);
+  },
+
+  play: function () {
+    window.addEventListener('keydown', this.onKeydown, false);
+  },
+
+  pause: function () {
+    window.removeEventListener('keydown', this.onKeydown);
+  },
+
+  onKeydown: function (evt) {
+    // Alt + Ctrl + e
+    var shortcutPressed = evt.keyCode === 69 && evt.ctrlKey && evt.altKey;

```

Comment:
we have [this helper function](https://github.com/aframevr/aframe/blob/f0c2b9f/src/utils/index.js#L126-L136). either use that, or check `document.activeElement` or make sure that `evt.target` is not a form field, `[contexteditable]` element, etc.